android/.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -1,5 +0,0 @@
-<component name="ProjectCodeStyleConfiguration">
```

Comment:
As I see more and more changes to this set of files I'd like to have it settled once and for a while 😉 I suggest changing this set of rules in separate PR.